src/components/charts/BarChart.tsx

```diff
@@ -386,12 +405,18 @@ const BarChart = () => {
       )}
       {isLoading && <Loader className="!p-0 m-auto min-h-[24rem]" />}
       {!isLoading && (
-        <div className="home-sessions flex justify-center basis-full relative px-3 py-8 sm:px-5 h-64 mb-4 bg-black-200 w-[full] h-[22rem] sm:h-[60vh] drop-shadow-md rounded-xl flex-col min-h-[24rem]">
+        <div
+          className={`${
+            location.pathname.includes("home")
+              ? "home-sessions"
+              : "sessions-main"
```

Comment:
keep the naming consistent.
either it is home-sessions and main-sessions OR
it is sessions-home and sessions-main.